### ISSUE TITLE
Email recover password email

### ISSUE THREAD

**Issue by Senen** [AyuntamientoMadrid/agendas#236]
What
====
Recovery password email is generated in English.

```
Hello senen@rockandror.com!

Someone has requested a link to change your password. You can do this through the link below.

Change my password

If you didn't request this, please ignore this email.

Your password won't change until you access the link above and create a new one.

```

How
===
Fix locale configurations or add missing translations

**Comment by raul-fuentes** [comment]
I'll do this one

**Comment by Senen** [comment]
Closed via #246. Thanks @raul-fuentes! 🤗
